app/celery/v3/notification_tasks.py

```diff
@@ -128,13 +130,23 @@ def v3_process_notification(request_data: dict, service_id: str, api_key_id: str
     return
 
 
-# TODO - retry conditions
-# TODO - error handling
+@notify_celery.task(
+    serializer="pickle",
+    autoretry_for=(AwsSesClientException,),
+    retry_backoff=True,
+    retry_backoff_max=3600,  # 1 hour
+    max_retries=26           # This is about 1 day total of retries.
```

Comment:
Please use the same retry times set with deliver_email.

Reply:
done